src/js/components/Tab/ViewOnlyTab.tsx

```diff
@@ -6,18 +6,27 @@ import TabContent from 'components/Tab/TabContent'
 import { TabProps } from 'components/types'
 import PIN from './Pin'
 import ContainerIndicator from './ContainerIndicator'
+import CloseButton from 'components/CloseButton'
 
 export default observer((props: TabProps) => {
   const { tab } = props
   const pin = tab.pinned && PIN
 
+  const onRemove = () => {
+    const { removing, remove } = tab
+    if (!removing) {
+      remove()
+    }
+  }
+
   return (
     <div tabIndex={-1} className='relative flex w-full'>
       {pin}
       <Icon tab={tab} faked />
       <TabContent tab={tab} faked />
       <TabTools tab={tab} faked />
       <ContainerIndicator cookieStoreId={tab.cookieStoreId} />
+      <CloseButton onClick={onRemove} disabled={tab.removing} />
```

Comment:
Could you please apply this patch: https://github.com/xcv58/Tab-Manager-v2/commit/16bdd1fb6248a210dae403ed8bebe67d0bdd9176

So that the search list wouldn't close after users click the close button?

Reply:
Updated. Added this commit:
https://github.com/xcv58/Tab-Manager-v2/pull/594/commits/b8974d395e22b3c824911aa776944d27de1f9441